modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -3546,6 +3546,12 @@ version 2.2.3.
 ### Changed
 - [LPS-79679]: Update the [Liferay Gradle Plugins] dependency to version 3.12.3.
 
+## 5.4.0 - 2018-05-15
+
+### Added
+- [LPS-80944]: Expose `LiferaySpringBootDefaultsPlugin` as a Gradle plugin with ID
```

Comment:
I'd just say something like "Add the plugin `com.liferay.spring.boot.defaults` to configure Spring Boot projects according to Liferay defaults."